crates/ffi/src/lib.rs

```diff
@@ -86,6 +107,16 @@ mod ffi {
 
         pub async fn get_address(&self) -> Result<String, FFIError>;
 
+        pub async fn prepare_sign_message(
+            &self,
+            _message_hash: String,
+        ) -> Result<FFIPreparedSignature, FFIError>;
+
+        pub async fn do_sign_message(
```

Comment:
just thinking what do we need this method for?

Reply:
I think we addressed this

Reply:
yep, maybe we can just add the 3rd method bindings

Reply:
That's `finalize_sign_message()`

Reply:
I can't see it

Reply:
It's in `crates/ffi/src/account_client.rs`

I didn't add to Swift